src/yang/ietf-bgp.yang

```diff
@@ -858,6 +858,54 @@ module ietf-bgp {
             "Clear neighbors action.";
 
           input {
+	    choice operation {
+	      default operation-admin;
+	      description
+	        "The type of operation for the clear action.";
+	      case operation-admin {
+	        leaf admin {
+		  type empty;
+		  description
+		    "Closes the Established BGP session with a BGP
+		     NOTIFICATION message with the Administrative
+		     Reset error subcode.";
+		  reference
+		    "RFC 4486.";
+		}
+	      }
+	      case operation-hard {
+	        leaf hard {
+		  type empty;
+		  description
+		    "Closes the Established BGP session with a BGP
+		     NOTIFICATION message with the Hard Reset error
+		     subcode.";
+		  reference
+		    "RFC 8538, Section 3.";
```

Comment:
Add the full reference here also.

Reply:
I've done so, but we're inconsistently documenting references.  4271 is cited everywhere for obvious reasons, but without similar expansion.